docs/langref/src/concepts/layouting.md

```diff
@@ -132,9 +132,9 @@ For more fine grained control, the `padding` property can be split into properti
 
 ## `VerticalLayout` and `HorizontalLayout`
 
-The `VerticalLayout` and `HorizontalLayout` elements place elements in a column or row.
-By default, they will be stretched or shrunk so that they take the whole space, and their
-alignment can be adjusted.
+The `VerticalLayout` and `HorizontalLayout` elements place elements in a column or a row.
+By default, they will be stretch or shrink so that they take the whole space. You
+can adjust the element's alignmnet as needed.
```

Comment:
```suggestion
By default, they will be stretched or shrunk so that they take the whole space. You
can adjust the element's alignment as needed.
```

or

```suggestion
By default, they stretch or shrink so that they take the whole space. You
can adjust the element's alignment as needed.
```